pom.xml

```diff
@@ -132,6 +132,12 @@
             <artifactId>mssql-jdbc</artifactId>
             <version>8.4.0.jre8</version>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <version>1.4.200</version>
```

Comment:
@KushnirykOleh -- could we switch the version to `1.4.199` instead so that it doesn't conflict with the `1.4.200` version which we are using in the LMS course?

Reply:
As it's jdbc driver version, not DB version, they are backward compatible. And usually have some improvements. In https://liquibase.talentlms.com/unit/view/id:2580 we mention this version to add if it's not present in pom.xml

Even having both
```
        <dependency>
            <groupId>com.h2database</groupId>
            <artifactId>h2</artifactId>
            <version>1.4.200</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>com.h2database</groupId>
            <artifactId>h2</artifactId>
            <version>1.4.200</version>
            <scope>test</scope>
        </dependency>
```

or 
```
        <dependency>
            <groupId>com.h2database</groupId>
            <artifactId>h2</artifactId>
            <version>1.4.200</version>
            <scope>test</scope>
        </dependency>
        <dependency>
            <groupId>com.h2database</groupId>
            <artifactId>h2</artifactId>
            <version>1.4.199</version>
            <scope>test</scope>
        </dependency>
```
will work

Reply:
sounds good @KushnirykOleh